### DISCUSSION TITLE
Screen Reader Accessibility MultiSelect

### DISCUSSION THREAD

**Discussion by tkurschner** [primefaces/community#748] [PrimeNG] [answered]
Hello Team,
I am using the NVDA screen reader. Whenever I click on one of the items, the screen reader will not read the item clicked in the MultiSelect. This is also replicated on all of these examples https://primeng.org/multiselect. Is there an open issue already being worked on for this? Thanks!

**Comment by SoyDiego** [comment, marked as answer]
> Hello Team, I am using the NVDA screen reader. Whenever I click on one of the items, the screen reader will not read the item clicked in the MultiSelect. This is also replicated on all of these examples https://primeng.org/multiselect. Is there an open issue already being worked on for this? Thanks!

Hi, you can check if the issue exists here: https://github.com/primefaces/primeng/issues
If not, you can create one and add all the information possible to replicate the problem, screenshots, etc...

Thanks! :)
